testgrid/config.yaml

```diff
@@ -4561,28 +4561,28 @@ dashboards:
   dashboard_tab:
   - name: gce-new-master-upgrade-master
     test_group_name: ci-kubernetes-e2e-gce-new-master-upgrade-master
-    description: Upgrade master only, in gce(gci), from 1.10 to master, run non-parallel-safe tests only
+    description: TODO: think this is owned by google-because-cluster-up-bash, set an alert up; TODO: what file does this come from; Upgrade master only, in gce(gci), from 1.10 to master, run non-parallel-safe tests only
```

Comment:
google-because-cluster-up-bash is my awful name for it, I would rather we use a group of people for this rather than specific e-mail addresses if at all possible